package.json

```diff
@@ -1,17 +1,23 @@
 {
   "name": "eslint-config-mavenlint",
-  "version": "1.0.3",
+  "version": "1.1.0",
   "description": "Mavenlink ESLint config",
   "main": "index.js",
   "repository": "https://github.com/mavenlink/mavenlint.git",
   "license": "MIT",
-  "peerDependencies": {
-    "eslint": ">= 3.7.0"
+  "devDependencies": {
+    "eslint": "^3.7.0",
+    "eslint-plugin-import": "^1.13.0",
+    "eslint-plugin-jsx-a11y": "^2.1.0",
+    "eslint-plugin-react": "^6.1.2"
   },
-  "dependencies": {
-    "eslint-config-airbnb": "^10.0.1",
+  "peerDependencies": {
+    "eslint": "^3.7.0",
```

Comment:
I recently learned that it is recommended that shareable configs use a more flexible version spec for their peer dependencies (see http://eslint.org/docs/developer-guide/shareable-configs#publishing-a-shareable-config).

If we want to follow that recommendation, this should say `>= 3.7.0` and the three lines below should also be changed.